cupsfilters/ipp.c

```diff
@@ -30,6 +31,7 @@
 #include <cups/cups.h>
 #include <cups/backend.h>
 #include <cups/dir.h>
+#include <cups/http-private.h>
```

Comment:
Do not use private (library-internal) APIs in other programs/libraries. They are not stable APIs and can change at any time.